README.md

```diff
@@ -245,7 +245,7 @@ You may obtain a copy of the License at
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
+See the [License](https://github.com/Kong/kong/blob/master/LICENSE) for the specific language governing permissions and
```

Comment:
A link here will not render (see https://github.com/Kong/kong/blob/e427c687944f809561825c0a20e14c7733d8be68/README.md). Also, we don't _have_ to have a link here. Lastly, the formatting broke and we are exceeding the 80 columns per line limit.